ibl_to_nwb/updated_conversion/datainterfaces/ibllickinterface.py

```diff
@@ -0,0 +1,37 @@
+from one.api import ONE
+from pydantic import DirectoryPath
+from pynwb import DynamicTable, H5DataIO
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.tools.nwb_helpers import get_module
+
+
+class IblLickInterface(BaseDataInterface):
+    def __init__(self, session: str, cache_folder: DirectoryPath):
+        self.session = session
+        self.cache_folder = cache_folder
+
+    def run_conversion(self, nwbfile, metadata: dict):
+        one = ONE(
+            base_url='https://openalyx.internationalbrainlab.org',
+            password='international',
+            silent=True,
+            cache_folder=self.cache_folder
+        )
+
```

Comment:
I'd pass this in the init, similar to the other classes

Reply:
Yep, that's the plan